From 209a62de64903f2d7f0b41270271e03121808ee5 Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <josef.schlehofer@nic.cz>
Date: Sun, 10 Jan 2021 14:08:08 +0100
Subject: [PATCH] turris1x-support: turris1x.dtb is in folder dts

Right now, compilation fails because of this:
install: cannot stat 'foo/build/build_dir/target-powerpc_8540_musl/linux-mpc85xx_p2020/linux-5.4.87/arch/powerpc/boot/turris1x.dtb'>: No such file or directory
That's because .dtb file is located in dts folder.

---
 hardware/turris1x/turris1x-support/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hardware/turris1x/turris1x-support/Makefile b/hardware/turris1x/turris1x-support/Makefile
index b5f53a544..94286d8c2 100644
--- a/hardware/turris1x/turris1x-support/Makefile
+++ b/hardware/turris1x/turris1x-support/Makefile
@@ -34,7 +34,7 @@ endef

 define Package/turris1x-support/install
 	$(INSTALL_DIR) $(1)/boot
-	$(INSTALL_DATA) $(LINUX_DIR)/arch/$(LINUX_KARCH)/boot/turris1x.dtb "$(1)"/boot/fdt
+	$(INSTALL_DATA) $(LINUX_DIR)/arch/$(LINUX_KARCH)/boot/dts/turris1x.dtb "$(1)"/boot/fdt
 	mkimage -T script -C none -n boot -d files/boot.txt "$(1)"/boot.scr
 endef

--
2.25.1
